src/NLog.Redis/NLog.Redis.csproj

```diff
@@ -14,17 +14,24 @@
     <PackageLicenseUrl>https://raw.githubusercontent.com/richclement/NLog.Redis/master/LICENSE</PackageLicenseUrl>
     <RepositoryType>git</RepositoryType>
     <RepositoryUrl>git://github.com/richclement/NLog.Redis</RepositoryUrl>
-    
+
+  </PropertyGroup>
+
+  <PropertyGroup Condition=" '$(Configuration)' == 'Release' ">
+    <SignAssembly>true</SignAssembly>
+    <AssemblyOriginatorKeyFile>../NLog.snk</AssemblyOriginatorKeyFile>
+    <DelaySign>false</DelaySign>
+    <PublicSign Condition=" '$(OS)' != 'Windows_NT' ">true</PublicSign>

```

Comment:
looks good, but when we do strong naming, we should fix the assembly version. 

So please add ` <AssemblyVersion>1.0.0.0</AssemblyVersion>` (and maybe an XML comment so we only update the AssemblyVersion on major versions)

Reply:
Would the assembly version differ than the Nuget package version? I wonder if handling this during building would be easiest if the assembly version is to be consistent (the csproj's can be patched during build).

Reply:
> Would the assembly version differ than the Nuget package version?

Yes! YES! See https://github.com/NLog/NLog/issues/1765#issuecomment-261776513

(and yes, it has downsides and gives sometimes questions, but IMO it's the best solution.  PS JSON.NET also uses this scheme)

Reply:
Ooh, I understand now - I get what you're going for, I'll make the needed changes. 

So the AssemblyVersion would be hard coded, but FileVersion, InformationalVersion, and PackageVersion would be patched?